pyemu/utils/helpers.py

```diff
@@ -2154,6 +2169,8 @@ def _process_list_file(model_file, df):
             common_idx = (
                 new_df.index.intersection(mlts.index).sort_values().drop_duplicates()
             )
+            if common_idx.shape[0] == 0:
```

Comment:
Ee might be a able to do better here. Perhaps we can pass the number of expected alignments to that tracker file and then check against that here? Testing something at my end now, but lets go with this for them being.

Reply:
agreed - explicitly tracking the original alignment would be much safer...